nextcord/interactions.py

```diff
@@ -636,6 +643,18 @@ async def send_message(
                 raise ValueError('embeds cannot exceed maximum of 10 elements')
             payload['embeds'] = [e.to_dict() for e in embeds]
 
+        if file is not MISSING and files is not MISSING:
+            raise TypeError('cannot mix file and files keyword arguments')
```

Comment:
```suggestion
            raise TypeError('Cannot mix file and files keyword arguments')
```

Reply:
Should I change it for embeds as well?

The checks were copied from embeds since they are similar